src/utils/typography.js

```diff
@@ -10,119 +10,138 @@ export const primaryFont = "tondo, sans-serif";
 //     button: '1.35em'
 // };
 
-
 export const H1 = styled.h1`
-    font-family: ${primaryFont};
-    font-size: 1.6em;
-    line-height: 1.3em;
-    letter-spacing: -0.017em;
-    color: ${props => props.color ? props.color : colors.primaryGray};
-    margin: ${props => props.margin ? props.margin : 0};
+  font-family: ${primaryFont};
+  font-size: 1.6em;
+  line-height: 1.3em;
+  letter-spacing: -0.017em;
+  color: ${(props) => (props.color ? props.color : colors.primaryGray)};
+  margin: ${(props) => (props.margin ? props.margin : 0)};
 `;
 
 export const H2 = styled.h2`
-    font-family: ${primaryFont};
-    font-size: 1.25em;
-    line-height: 1.05em;
-    letter-spacing: -0.018em;
-    font-weight: ${props => {
-        switch (props.wight) {
-            case "light":
-                return "100"
-            case "regular":
-                return "400"
-            case "bold":
-                return "700"
-            default:
-                return "400"
-        }
-    }};
-    color: ${props => props.color ? props.color : colors.primaryGray};
-    margin: ${props => props.margin ? props.margin : 0};
+  font-family: ${primaryFont};
+  font-size: 1.25em;
+  line-height: 1.05em;
+  letter-spacing: -0.018em;
+  font-weight: ${(props) => {
+    switch (props.wight) {
+      case "light":
+        return "100";
+      case "regular":
+        return "400";
+      case "bold":
+        return "700";
+      default:
+        return "400";
+    }
+  }};
+  color: ${(props) => (props.color ? props.color : colors.primaryGray)};
+  margin: ${(props) => (props.margin ? props.margin : 0)};
```

Comment:
The margin must be consistent. It shouldn't be use as props

Reply:
they are not consistent in the design

Reply:
Yes, and this is problematic, the margins must be consistent. 
As said before, the purpose of using styled-components and working on a component library is to make things clear and consistent. 

If there are different types for one element, then, use modifiers, everything must be consistent, it should not be defined every time you use the component, otherwise there is no advantage in creating a component library.